charts/minecraft-bedrock/values.yaml

```
@@ -29,7 +29,7 @@ minecraftServer:
   # This must be overridden, since we can't accept this for the user.
   eula: "FALSE"
   # One of: LATEST, SNAPSHOT, or a specific version (ie: "1.7.9").
-  version: "1.14.4"
+  version: "1.16.1.02"
```

Actually this would be a good default:
```suggestion
  version: "LATEST"
```

---

Agreed, will change this!